app/controllers/RestApi.scala

```diff
@@ -260,7 +253,7 @@ class RestApi @Inject() (
     taskDao.find(id).map(task => Ok(Json.toJson(task)))
   }}
 
-  def createTask() = Action.async(validateJson[Task]) {implicit request => {
+  def createTask() = Action.async(validateJson[TaskDB]) { implicit request => {
```

Comment:
Instead of using the database specific implementation `TaskDB` we should use a business model implementation `Task`.

Reply:
#123